### ISSUE TITLE
Deploy Questions

### ISSUE THREAD

**Issue by miggall** [CDLUC3/dmptool#432]
**What version of the DMPRoadmap code are you running? (e.g. v2.2.0)**
v4.0.5

Curious about the "deploy.rb" configuration. I can see that you are bundle installing without a few groups
`set :bundle_without, %w[pgsql thin rollbar test].join(' ')`
however it appears the development group is still installed? Are you running rails spring in production? Or are you simply installing the gem and disabling spring by setting the `DISABLE_SPRING` environment variable?

I was attempting to get a slimmed production build but excluding the `development` group from the bundle caused errors. Any suggestions, advice?

**Comment by briri** [comment]
Hi @miggall i haven’t looked at the deploy script in quite some time. My guess is that some of the gems in the development group should probably not be because they’re needed in all environments. 

We manage the app via systemd using the puma commands. We don’t have the DISABLE_SPRING var set so there may be some unnecessary stuff getting loaded. 

I’d be very interested to see what you end up with. We’ve been dealing with some memory issues lately #419 and any improvements in this area would likely help. 

I’m off this week but will have a look at our current prod setup when I’m back

**Comment by miggall** [comment]
@briri alright, thank you! I was curious how you all were going about it. I ended up containerizing the entire application for deployment and I as I started to slim down and optimize I started getting running into issues mainly with spring. This does alleviate some of my concerns, however. I'll keep the issue open only to get some feedback from you when you return but otherwise I feel you have answered my question.
